src/OpenLoco/Station.cpp

```diff
@@ -779,4 +776,27 @@ namespace OpenLoco
 
         setStationCatchmentRegion(cargoSearchState, minPos, maxPos, flag);
     }
+
+    string_id getTransportIconsFromStationFlags(const uint16_t flags)
+    {
+        static const string_id label_icons[] = {
```

Comment:
Any particular reason the `constexpr` was removed?

Reply:
There wasn't much point in it after moving it to the cpp file. Guess it doesn't really matter when in the cpp file.

Reply:
I've reverted it back to a constexpr